fission/src/mirabuf/MirabufSceneObject.ts

```diff
@@ -49,6 +47,8 @@ class MirabufSceneObject extends SceneObject {
     private _ejectable?: EjectableSceneObject
     private _scoringZones: ScoringZoneSceneObject[] = []
 
+    private _gizmo: GizmoSceneObject | undefined
```

Comment:
Since we've decided that only once gizmo will exist, this should be some kind of static either in the SceneRenderer or GizmoSceneObject itself.

Reply:
I can think of a case where we would want more than one gizmo in a scene at a time. For instance, what about placing more than one robot during the initialization of match mode?

Reply:
Honestly, I think for initial robot placement, we should move away from using the gizmo tool altogether and move towards Luca's placement for bots for match mode in v6 where you can click and place. In which case, gizmo controls should be reserved for fine movement/orientation control.